XcodeProj/Examples/Swift/CustomViewController.swift

```diff
@@ -17,7 +17,7 @@ class CustomViewController: UIViewController, MGLMapViewDelegate {
     var userRoute: Route?
 
     // Start voice instructions
-    let voiceController = RouteVoiceController()
+//    let voiceController = MapboxVoiceController()
```

Comment:
blijft dit commented? 🤔

Reply:
MapboxSpeech has been removed from the library before. This is a leftover.

Reply:
If you don't mind, it would be great if you could revert this change. The RouteVoiceController should work without any Mapbox token.

Reply:
Reverted 👌